azure/functions/decorators/function_app.py

```diff
@@ -1071,3 +1073,134 @@ def decorator():
             return decorator()
 
         return wrap
+
+    def on_blob_change(self,
+                       arg_name: str,
```

Comment:
Is there a plan for adding a default value for arg_name? @YunchuWang - did we add default arg name values for other triggers/bindings?

Reply:
Arg name does not have a default value.. We can definitely add a default value later one we have our templates ready.